py/bodynoline.py

```diff
@@ -1,19 +1,20 @@
 import dominate
 from dominate.tags import *
 
-def _bodynoline():
-    """Returns a body section with no vertical lines. TODO: edit this description
-
-    TODO: A description here, according to Google's Python style guide https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
+def _bodynoline(title, entries):
+    """Returns the HTML for a centered body section with no vertical lines. The section has a 
+    title, such as Alumni Advice or Member Spotlight, as well as a body section that contains
+    text along with images.
 
     Args:
-        TODO
+        title (string) - The title of the newsletter section.
```

Comment:
Maybe add an example of a title and entries that are passed in

Reply:
Good idea!